### ISSUE TITLE
Icon Request: equal housing opportunity

### ISSUE THREAD

**Issue by rburch** [FortAwesome/Font-Awesome#3574]
I've needed this a couple times in the past year for realtor websites.
http://www.historicroundhillsummit.com
http://www.westsquareboston.com/
![equal_housing](https://cloud.githubusercontent.com/assets/1537395/3116463/52e66d9a-e71a-11e3-90f7-561540078f11.jpg)

**Comment by Tuckie** [comment]
+1

**Comment by estebansanti** [comment]
+400

Would be great to have this!

**Comment by ChristianYaglenski** [comment]
+1

**Comment by adambundy** [comment]
+1

**Comment by krsbrn** [comment]
+10000000

**Comment by elvismdev** [comment]
+2!

**Comment by micah1701** [comment]
+1

**Comment by jwrussell83** [comment]
+1

**Comment by sharksharkco** [comment]
+1

**Comment by patmcneil** [comment]
+1

**Comment by aaronkaiser** [comment]
+1

**Comment by kamenlee** [comment]
+1

**Comment by rwilsond** [comment]
![upvote-here](https://cloud.githubusercontent.com/assets/12199142/17336035/6b6e91fe-58aa-11e6-8124-6cf6764f087c.jpg)

**Comment by antonioortegajr** [comment]
+1

**Comment by bryandowd** [comment]
+1

**Comment by krsbrown** [comment]
PLEASE PLEASE PLEASE

**Comment by ausmurp** [comment]
This is pretty old. Any update on this?

**Comment by cotillardq** [comment]
+1

**Comment by sharksharkco** [comment]
We have a pied piper logo and still not this, they provide the files!
https://portal.hud.gov/hudportal/HUD?src=/library/bookshelf11/hudgraphics/fheologo

**Comment by krsbrown** [comment]
I'd like to see a version without the text under it too. Just the symbol
itself.

**Comment by sharksharkco** [comment]
@krsbrown I agree, I think that's how they should do it. I don't think it's a requirement to have the text but people can add it themselves if they like

**Comment by rwilsond** [comment]
![image](https://user-images.githubusercontent.com/12199142/39645246-9e3327a0-4fa5-11e8-8dd7-3c6e5ca2a1f3.png)

**Comment by Ednas** [comment]
+1

**Comment by myonlyclone** [comment]
+1 and EOE and EHL

**Comment by krsbrown** [comment]
I need this for almost every project we do. What is taking so long to get this as an option? We have the pro account, Is it possible to add a custom icon for our use?

**Comment by bhubbard** [comment]
@krsbrown Font Awesome gets a lot of requests. I know you can speed up more serious requests by paying to get them into the project sooner, you would need to contact them directly about that.

**Comment by shaneedwardtaylor** [comment]
I need the exact same thing. Would make things a lot easier if this were a font icon :)

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
